ajax/libs/jQueryNearestElement/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "jQueryNearestElement", 
+  "description": "This plugin finds the elements in a page that are closest to a particular point or element, based on pixel dimensions. ",
+  "keywords": [
+    "jquery-plugin", 
+    "ecosystem:jquery", 
+    "nearest", 
+    "furthest", 
+    "touching", 
+    "pixel", 
+    "distance"
+  ],
+  "author": "Gilmore Davidson", 
+  "homepage": "https://gilmoreorless.github.io/jquery-nearest/",
+  "repository": {
+    "type": "git",  
+    "url": "git+https://github.com/gilmoreorless/jquery-nearest.git"
+  },
+  "filename": "jquery.nearest.js",
+  "npmName": "jquery-nearest",
+  "npmFileMap": [
+    {
+      "basePath": "src",
```

Comment:
Let's use git auto-update because there are more release versions from [GitHub](https://github.com/gilmoreorless/jquery-nearest/releases)(12) than npm(2), so let choice git auto-update. This will let the web developers choose more versions.

Let's modify from A to B:
- A
```js
  "npmName": "jquery-nearest",
  "npmFileMap": [
    {
      "basePath": "src",
      "files": [
        "**/*"        
      ]
    }
  ],
```

- B
```js
  "autoupdate": {
    "source": "git",  
    "target": "git://github.com/gilmoreorless/jquery-nearest.git",
    "fileMap": [
      {
        "basePath": "src",
        "files": [	
            "jquery.nearest*"
        ]
      }
    ]
  },
```